extensions/wikia/UserLogin/js/UserSignupAjaxValidation.js

```diff
@@ -9,12 +9,15 @@
 	 * - wikiaForm: instance of WikiaForm
 	 * - inputsToValide: array of input names to be ok'ed before submission
 	 * - submitButton: pointer to main submit button of the form
+	 * - passwordInputName: input name attribute for form's password input
 	 * @constructor
 	 */
 	var UserSignupAjaxValidation = function (options) {
 		this.wikiaForm = options.wikiaForm;
 		this.inputsToValidate = options.inputsToValidate || [];
```

Comment:
Oh yeah, I got rid of the functions that were using this array. Looks like we can get rid of that option too.